src/server.ts

```diff
@@ -150,6 +152,17 @@ function browsersListParser(data: string): IBrowsersList {
 		.map((line: string) => line.trim());
 }
 
+function getDocumentPath(document: string): string {
+	return document
```

Comment:
Can we rewrite this to use the standard `path` module with `path.relative` or `path.resolve`?

Reply:
Good point! Replaced!